### DISCUSSION TITLE
adding custom CSS to TinyMCE editor?

### DISCUSSION THREAD

**Discussion by atomGit** [GetPublii/Publii#1840] [Help]
the title of a post in the WYSIWYG editor is much larger than it needs to be - this is a waste of editor space (#1305) - i'm familiar with Electron's dev tools, but i never figured out where the custom css file needs to be placed to override the default styling

for example, trying to change `.post-editor-form #post-title` font size in `/Publii/themes/simple/assets/css/editor.css` doesn't work

**Comment by candidexmedia** [comment]
Maybe this would help?

https://getpublii.com/dev/post-editor-api/

https://gist.github.com/candidexmedia/367711a80274b4c005fd5983df97d20c
